data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/export/ExportTaskManager.java

```diff
@@ -64,9 +73,13 @@ public String getExportManifest(String exportArn) {
 
         String manifestKey = null;
         try {
+            dynamoAggregateMetrics.getExportApiInvocations().increment();
             DescribeExportResponse resp = dynamoDBClient.describeExport(request);
             manifestKey = resp.exportDescription().exportManifest();
 
+        } catch (final InternalServerErrorException e) {
+            dynamoAggregateMetrics.getExport5xxErrors().increment();
+            LOG.error("Unable to get manifest file for export " + exportArn);
```

Comment:
Should we log the error here too? Same applies for the `catch` section below

Reply:
Ah yeah we should log that